### PR TITLE
Tools of the trade: How I communicate at GitHub

### PR DESCRIPTION
**Pull request by benbalter**
-----
[View rendered _posts/2020-08-10-tools-of-the-trade.md](https://github.com/benbalter/benbalter.github.com/blob/tools-of-the-trade/_posts/2020-08-10-tools-of-the-trade.md)

### CODE REVIEW

File: _posts/2020-08-10-tools-of-the-trade.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+title: "Tools of the trade: How I communicate at GitHub"
+description:
+---
+
+### Principles
```
Comment (corywilkerson) [inline _posts/2020-08-10-tools-of-the-trade.md]: Meta-principle is being principled about how you communicate.

This is a great list. May not be part of your core philosophy but 'fewer places to look' has been really useful (or at least knowing where to look.) Even if you have all the 'how' nailed the 'where' becomes problematic.

File: _posts/2020-08-10-tools-of-the-trade.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#### Public (to the company) channels
+
+* **When**: Questions, updates, coordination, and to amplify messages communicated elsewhere. This should be your default method of chat unless DMs or a private channel are required.
+* **Who**: Team, stakeholders, the entire organization.
+* **Why**: Discoverability, passive awareness, and learning.
+
+### Documentation (Git)
+
+* **Who**: Team, stakeholders, the entire organization
+* **When**: Project plans, technical overviews, ADRs, team process, etc. - anything non-ephemeral and without state
```
Comment (corywilkerson) [inline _posts/2020-08-10-tools-of-the-trade.md]: Here you say 'without state' whereas I see repos as state 'capture.'

File: _posts/2020-08-10-tools-of-the-trade.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+* Default to open, visible, and discoverable
+* Prefer asynchronous, unless the conversation requires higher-fidelity
+* Use systems that naturally capture and expose process. If not, memorialize after the fact.
+* Use the best tool for the job, but favor to GitHub
+* Err on the side of over-communicating
+* Write things down and capture the "why"
+* [Everything should have a URL](https://ben.balter.com/2015/11/12/why-urls/)
+
+### Google Docs
```
Comment (corywilkerson) [inline _posts/2020-08-10-tools-of-the-trade.md]: Feels like Google Docs deserves a small caveat emptor or caution label though I think you mostly capture it with your 'small groups' note. TL;DR Google Docs discovery is very, very poor (as is understanding the state of a thing even if you find it.)

File: _posts/2020-08-10-tools-of-the-trade.md (hunk 2 above)
Comment (benbalter) [reply _posts/2020-08-10-tools-of-the-trade.md]: Interesting. I was thinking more in terms of ██████ our company-wide documentation repo. Perhaps it's a matter of clarifying that "without state" is really without open/closed state, but that it's primarily used to capture the current state of things?

**Pull request merged**